cves/CVE-2012-6617.yml

```
@@ -146,10 +144,10 @@ unit_tested:
     In FFmpeg, they have the FATE suite of regression tests. They are not
     "unit" tests per se, but are automated regression tests. If the fix for the
     vulnerability involved adding or updating a FATE suite, then make fix: true.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: true
+  code_answer: There were a lot of unit tests that covered the server of FFmpeg
+  fix: true
+  fix_answer: This could be solved if they used the correct vector that would cause a DOS.
```

```suggestion
  fix_answer: This could be solved if they used a specific vector that would cause a denial of service attack.
```
The wording just seemed a bit off to me, and I expanded the acronym  for the kiddies reading and learning this